data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -5304,6 +5304,41 @@ spec:
                       This resource group must be empty with no other resources when trying to use it for creating a cluster.
                       If empty, a new resource group will created for the cluster.
                     type: string
+                  subnets:
+                    description: Subnets is the list of subnets the user can bring
+                      into the cluster to be used.
+                    items:
+                      description: SubnetSpec specifies the properties the subnet
+                        needs to be used in the cluster.
+                      properties:
+                        cidr:
+                          description: |-
+                            SubnetCIDR specifies the CIDR for the subnet if it needs to be created.
+                            Should not be mentioned if the subnet is a bring your own subnet.
+                          items:
+                            type: string
+                          type: array
+                        name:
+                          description: Name of the subnet.
```

Comment:
`name` is mandatory. 
1. if a user specifies `name` and `cidr`, Installer needs to create a subnet of a specific name with the specified `cidr` ?
2. if a user specifies `name` and not the `cidr`, we conclude that this is a pre-created subnet and has to be validated for its presence?